source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -35,13 +42,21 @@ partitions = this would theoretically provide you up to: ``~896,000
 PUT/POST/DELETE TPS and 1,408,000`` GET TPS if requests are distributed evenly
 across the partitions).
 
+Configuration
+^^^^^^^^^^^^^
+
 .. code-block:: none
 
   plugin {
-    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
-    obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_index_fs = compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
   }
 
+.. Note::
+  If the version you are using does not support the ``aws-s3`` scheme, try using the more generic ``s3`` scheme.
```

Comment:
I just noticed that some of the lines in this file are line wrapped while others aren't.  I don't know what the preferred format is, but I'd say at least the lines you're touching should be properly wrapped/not-wrapped.

Reply:
I cleaned up a bit so at least the things i changed on this page are wrapped now.